package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@yext/answers-search-ui",
-  "version": "1.7.3",
+  "version": "1.7.4",
```

Comment:
I think we need to update the version in static/package.json as well! (I forgot this a number of times in the past)